extensions/api/observability/src/main/java/org/eclipse/dataspaceconnector/api/observability/ObservabilityController.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2021 Daimler TSS GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Daimler TSS GmbH - Initial API and Implementation
+ *       Fraunhofer Institute for Software and Systems Engineering - add negotiation endpoint
+ *
+ */
+
+
+package org.eclipse.dataspaceconnector.api.observability;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.dataspaceconnector.spi.system.readiness.ReadinessService;
+
+
+@Consumes({ MediaType.APPLICATION_JSON })
+@Produces({ MediaType.APPLICATION_JSON })
+@Path("/")
+public class ObservabilityController {
+
+
+    private final ReadinessService readinessService;
+
+    public ObservabilityController(ReadinessService provider) {
+        readinessService = provider;
+    }
+
+    @GET
+    @Path("health")
+    public Response checkHealth() {
+        return getLiveness();
+    }
+
+    @GET
+    @Path("liveness")
+    public Response getLiveness() {
+        return readinessService.isLive() ?
+                Response.ok().build() :
```

Comment:
consider returning a body, e.g. from Spring Actuator: `{"status":"UP","components":{"db":{"status":"UP","details":{"database":"PostgreSQL","validationQuery":"isValid()"}},"diskSpace":{"status":"UP","details":{"total":62725623808,"free":48520187904,"threshold":10485760,"exists":true}},"readinessState":{"status":"UP"}}}`

Reply:
wouldn't that leak internal information?

Reply:
Health information should be posted on a separate port that is not accessible externally. With Jetty we can do this with `SelectChannelConnector`s.